cmd/commands/install.go

```diff
@@ -47,32 +49,53 @@ func installCmd(ctx *cli.Context) error {
 		return installAll()
 	}
 
+	args := ctx.Args()
 	sdkArg := ctx.Args().First()
 	if sdkArg == "" {
 		return cli.Exit("sdk name is required", 1)
 	}
-	argArr := strings.Split(sdkArg, "@")
-	argsLen := len(argArr)
+
 	manager := internal.NewSdkManager()
 	defer manager.Close()
-	if argsLen > 2 {
-		return cli.Exit("sdk version is invalid", 1)
-	} else {
-		var name string
-		var version internal.Version
-		if argsLen == 2 {
-			name = strings.ToLower(argArr[0])
-			version = internal.Version(argArr[1])
+
+	for i := 0; i < args.Len(); i++ {
+		sdkArg := args.Get(i)
+		argArr := strings.Split(sdkArg, "@")
+		argsLen := len(argArr)
+
+		if argsLen > 2 {
+			return cli.Exit(fmt.Sprintf("Your input(%s) is invalid", sdkArg), 1)
 		} else {
-			name = strings.ToLower(argArr[0])
-			version = ""
-		}
-		source, err := manager.LookupSdkWithInstall(name)
-		if err != nil {
-			return err
+			var name string
+			var version internal.Version
+			if argsLen == 2 {
+				name = strings.ToLower(argArr[0])
+				version = internal.Version(argArr[1])
+			} else {
+				name = strings.ToLower(argArr[0])
+				version = ""
+			}
+			source, err := manager.LookupSdkWithInstall(name)
+			if err != nil {
+				return err
+			}
+			err = source.Install(version)
+			if errors.Is(err, internal.ErrNoVersionProvided) {
+				// show prompt to let user select version
+				showAvailable, err := printer.Prompt("No version provided, do you want to select a version to install?")
```

Comment:
You can use [pterm interactive_confirm](https://github.com/pterm/pterm/tree/master/_examples/interactive_confirm)

Reply:
thanks for point that, looks it's a better replacement

Reply:
You can use it directly without encapsulation. After making a selection, the chosen result will be displayed automatically without the need to output it again.
Additionally, the `with*` methods can be used to set certain properties.